v2/datastream-to-spanner/pom.xml

```diff
@@ -80,6 +80,13 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+<!--        TODO - Remove after https://github.com/apache/beam/issues/30700 is fixed. -->
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-spanner</artifactId>
+            <version>6.61.0</version>
```

Comment:
Initially, I used 6.62.0 but after a discussion with @arawind I came to know that in order to use 6.62.0 it needs to available in google3 as well. And since FLOAT32 is available in 6.61.0 (client lib) we are using only that dependency to be safe that everything works seamlessly. @arawind Please comment if I have missed anything.